pkg/yang/entry.go

```diff
@@ -142,6 +142,13 @@ type ListAttr struct {
 	OrderedBy   *Value // order of entries determined by "system" or "user"
 }
 
+func NewListAttr() *ListAttr {
```

Comment:
Can we add a documentation comment here please? It should likely explain that this gives a 0->max value so that users know that this is the case.

Reply:
Done. Also changed name to `NewDefaultListAttr` as suggested elsewhere.